ui/src/main.rs

```diff
@@ -78,12 +80,13 @@ pub static STATIC_ARGS: Lazy<StaticArgs> = Lazy::new(|| {
     StaticArgs {
         uplink_path: uplink_path.clone(),
         cache_path: uplink_path.join("state.json"),
+        mock_cache_path: uplink_path.join("mock-state.json"),
         config_path: uplink_path.join("Config.json"),
         warp_path: uplink_path.join("warp"),
         logger_path: uplink_path.join("debug.log"),
         typing_indicator_refresh: 5,
         typing_indicator_timeout: 6,
-        use_mock: !args.no_mock,
+        use_mock: args.no_mock,
```

Comment:
this is a bug. it doesn't make mock be off by default. it inverts the meaning of the `--no-mock` flag.